app/controllers/game.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+const Game = mongoose.model('Game');
+
+/**
+ * start game if request is authenticated
+ * @return {void}
+ * @param {string} req - request from client end.
+ * @param {string} res - response send to client end.
+ */
+
+exports.start = (req, res) => {
+  const newGame = new Game();
+  // game_id should use _id
+  // this is just a sample data, should be updated for real user
```

Comment:
no need to use dummy data, you can construct the game data fro the request object
e.g 
```
const gameOnwer_id = req.body.gameOwnerId;
and the rest.
```